src/MoBi.Core/Services/ModuleFactory.cs

```diff
@@ -20,6 +21,12 @@ public ModuleFactory(IObjectBaseFactory objectBaseFactory, IObjectTypeResolver o
          _objectTypeResolver = objectTypeResolver;
       }
 
+      public Module CreateModuleWithName(string moduleName)
+      {
+         var newModule = _objectBaseFactory.Create<Module>().WithName(moduleName);
```

Comment:
Return directly instead of creating the variable. You can also change to an expression body when the method has only one line.